File: src/main/scala/no/hamre/polet/modell/ProductRelease.scala
package no.hamre.polet.modell

import java.time.LocalDate

case class ProductRelease
(
  releaseDate: LocalDate,
  products: List[MiniProduct],
  links: List[Link]
)

case class MiniProduct
(
  id: Long,
  varenummer: String,
  varenavn: String,
  volume: Double,
  price: Double,
  links: List[Link]
)

case class Link
(
  rel: String,
  href: String
)

File: backend/src/main/kotlin/no/hamre/polet/modell/ProductRelease.kt
package no.hamre.polet.modell

import java.time.LocalDate


data class ProductRelease
(
    val releaseDate: LocalDate,
    val products: List<MiniProduct>,
    val links: List<Link>
)

data class MiniProduct
(
    val id: Long,
    val varenummer: String,
    val varenavn: String,
    val volume: Double,
    val price: Double,
    val links: List<Link>
)

data class Link
(
    val rel: String,
    val href: String
)
